.github/workflows/test.yml

```diff
@@ -0,0 +1,16 @@
+name: NPM Test
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Set up NodeV14
```

Comment:
This step can probably be left out as you're not testing on a matrix, and GitHub Actions already has a recent node.js version preinstalled. I don't think we've explicitly documented which exact version of node.js is needed to run the tests.